Bobcat/Profile.cpp

```diff
@@ -369,9 +413,7 @@ void Profiles::Activate()
 {
 	if(int i = list.GetCursor(); i >= 0) {
 		Vector<Value> v = list.GetLine(i);
-		list.Remove(i);
-		list.Insert(0, v);
-		list.SetCursor(0);

```

Comment:
This slightly changed the existing functionality. I will add some better indication of current (and also default) profile soon.